wiki/Client/Interface/Cookie/en.md

```diff
@@ -2,6 +2,7 @@
 stub: true
 tags:
   - logo
+no_native_review: true
```

Comment:
Not required here. This tag is used for translations if there's no fluent person in the respective language available while a PR is reviewed.

```suggestion
```